tests/test_model_content_order.py

```diff
@@ -0,0 +1,10 @@
+from .utils import run_check, load_fixture_file
+
+
+def test_model_content_order_succeeds():
+    code = load_fixture_file('model_content_order.py')
+    issues = list(map(lambda x: x[2], run_check(code)))
```

Comment:
No need to call list or map here, let's just access the issues using indexes on the asserts:

```
issues = run_check(code)
assert 'DJ09' in issues[0][2]
```